common/test/acceptance/tests/video/test_studio_video_module.py

```diff
@@ -341,13 +341,18 @@ def setUp(self):
             super(CMSVideoA11yTest, self).setUp()
 
     def test_video_player_a11y(self):
-        # Limit the scope of the audit to the video player only.
-        self.outline.a11y_audit.config.set_scope(include=["div.video"])
+        self._create_course_unit(subtitles=True)
+        self.video.wait_for_captions()
+        self.assertTrue(self.video.is_captions_visible())
+
+        # limit the scope of the audit to the video player only.
+        self.outline.a11y_audit.config.set_scope(
+            include=["div.video"],
+            exclude=["a.ui-slider-handle"]
+        )
         self.outline.a11y_audit.config.set_rules({
             "ignore": [
-                'link-href',  # TODO: AC-223
+                # 'link-href',  # TODO: AC-223
```

Comment:
Since this isn't ignoring anything anymore, the entire `self.outline.a11y_audit.config.set_rules` call should be removed here.